src/supervisor.adoc

```diff
@@ -2021,6 +2021,11 @@ or VU-mode, or to execute SINVAL.VMA in VU-mode, raises a
 virtual-instruction exception. When `hstatus`.VTVM=1, an attempt to execute
 SINVAL.VMA in VS-mode also raises a virtual instruction exception.
 
+Attempting to execute SFENCE.W.INVAL or SFENCE.INVAL.IR in U-mode or VU-mode
+raises an illegal-instruction exception.
```

Comment:
Should raise a virtual-instruction in VU-mode as these instructions are HS-qualified.

Reply:
Good catch.  Otherwise looks good to me.

Reply:
This is what code review is for!